examples/plot_pitfalls_permutation_importance.py

```diff
@@ -0,0 +1,328 @@
+"""
+Pitfalls of Permutation Feature Importance (PFI) on the California Housing Dataset
+==================================================================================
+
+In this example, we illustrate the pitfalls of using permutation feature importance
+(PFI) on the California housing dataset.
+PFI measures the importance of a variable. However, it does not measure conditional
+importance and does not provide statistical control over the risk of making false
+discoveries, i.e., the risk of declaring a variable as important when it is not.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+from matplotlib.lines import Line2D
+from scipy.stats import ttest_1samp
+from sklearn.base import clone
+from sklearn.compose import TransformedTargetRegressor
+from sklearn.datasets import fetch_california_housing
+from sklearn.linear_model import RidgeCV
+from sklearn.metrics import r2_score
+from sklearn.model_selection import KFold, train_test_split
+from sklearn.neural_network import MLPRegressor
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import StandardScaler
+
+from hidimstat import CPI, PermutationImportance
+from hidimstat.conditional_sampling import ConditionalSampler
+
+#############################################################################
+# Load the California housing dataset and add a spurious feature
+# ------------------------------------------------------------------
+# The California housing dataset is a regression dataset with 8 features. We add a
+# spurious feature that is a linear combination of 3 features plus some noise.
+# The spurious feature does not provide any additional information about the target.
+
+dataset = fetch_california_housing()
+X_, y_ = dataset.data, dataset.target
+# only use 2/3 of samples to speed up the example
+X, _, y, _ = train_test_split(X_, y_, test_size=0.6667, random_state=0, shuffle=True)
+rng = np.random.RandomState(0)
```

Comment:
Can you put this line with the import?

Reply:
Well, it's an array of floats, I'm not sure it would be so insightful. However, in the example, [fetch_california_housing](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.fetch_california_housing.html#sklearn.datasets.fetch_california_housing) is linked to the description of the dataset in sklearn's doc.